apps/dashboard/src/components/variable/edit-variable-popover.tsx

```diff
@@ -20,20 +49,333 @@ export const EditVariablePopover = ({
   onUpdate,
   isAllowedVariable,
 }: EditVariablePopoverProps) => {
+  const isEnhancedDigestEnabled = useFeatureFlag(FeatureFlagsKeysEnum.IS_ENHANCED_DIGEST_ENABLED);
```

Comment:
all the code here was moved from the `EditVariablePopoverContent` which I removed; I needed to do this to have better control of the state and handling esc, enter, update; so these last three is a new functionality